src/components/notFound.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+
+export default function NotFound () {
+  return (
+    <div className='row not-found-page'>
+      <div className="not-found-page__text">
+        <h1 className='not-found-page__title'>
+          <b className='not-found-page__status-code'>404</b>
+          Page not found
+        </h1>
+        <div className='not-found-page__description'>
+          If you see this page, probably you have typed the incorrect address.
```

Comment:
"If you see this page, probably you have typed **an** incorrect address."
or
"If you see this page, probably you have typed the address incorrectly."